tracing-futures/src/lib.rs

```diff
@@ -85,6 +88,13 @@ pub trait WithSubscriber: Sized {
             dispatch: subscriber.into(),
         }
     }
+    fn with_current_subscriber(self) -> WithDispatch<Self> {
+        WithDispatch {
+            inner: self,
+            dispatch: dispatcher::get_default(|default| default.clone()),
```

Comment:
if we add a `Default` impl to `Dispatch` later, this could use that. I'd be happy to accept a PR that makes that change as well, but I'm also happy to merge this as-is.